requests/adapters.py

```diff
@@ -202,7 +202,8 @@ def cert_verify(self, conn, url, verify, cert):
 
         :param conn: The urllib3 connection object associated with the cert.
         :param url: The requested URL.
-        :param verify: Whether we should actually verify the certificate.
+        :param verify: Whether we should actually verify the certificate
+            or it could be a path to a ca bundle
```

Comment:
So, this docstring needs to be reworded. Let's have it be: `Either a boolean, in which case it controls whether we verify the server's TLS certificate, or a string, in which case it must be a path to a CA bundle to use`.